scripts/configs/active_sampler_learning.yaml

```diff
@@ -6,68 +6,68 @@ APPROACHES:
     FLAGS:
       explorer: "active_sampler"
       active_sampler_explore_task_strategy: "planning_progress"
-  competence_gradient_explore_ucb:
-    NAME: "active_sampler_learning"
-    FLAGS:
-      explorer: "active_sampler"
-      active_sampler_explore_task_strategy: "competence_gradient"
-      active_sampler_explore_use_ucb_bonus: True
+  # competence_gradient_explore_ucb:
+  #   NAME: "active_sampler_learning"
+  #   FLAGS:
+  #     explorer: "active_sampler"
+  #     active_sampler_explore_task_strategy: "competence_gradient"
+  #     active_sampler_explore_use_ucb_bonus: True
```

Comment:
can you revert all changes to this file just for the PR? Don't want to keep having this change on master every time we want a slightly different experiment

Reply:
yeah thanks!